src/tests.rs

```diff
@@ -401,6 +543,16 @@ async fn test_rpm_builder_async() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+// #[test]
+// fn test_rpm_builder_nofiles() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
This test fails currently because the implementation expects these tags to always be present.

Reply:
Should we expect that? I think we should support marker packages too, so I'd prefer if this is not disabled, but fixed.

Reply:
I'm just pointing out that we ought to do for the other iterators what we have done for the dependency-getters, and return empty vecs, but I'd like feedback on the approach before copying it over.  Agree that it's a bug that should be fixed